src/render.ts

```diff
@@ -58,6 +58,21 @@ export const formatRatio = (
     maximumFractionDigits,
   });
 
+export const isOverBudget = (bundle: {name: string, ratio: number}, budgetList: { name: string, budget: number}[]): string | null | undefined => {
```

Comment:
**suggestion**: Some of the functionality here is similar to the one in `diff.ts` for finding the budget for a specific chunk. And in both places there's an important nuance: we will use the _first_ budget for a given chunk. (I understand that is primarily a defensive measure against duplicates in the action input.)

Could we separate the logic of finding the budget entry for a specific chunk into its own function? Then we can have two "rendering" functions that render to the console, or to markdown.

Reply:
I have refactored by on the pattern that the diff webpack code